raiden/transfer/mediated_transfer/initiator.py

```diff
@@ -23,13 +23,41 @@
 )
 from raiden.transfer.state import (
     CHANNEL_STATE_OPENED,
-    message_identifier_from_prng,
     RouteState,
     NettingChannelState,
+    message_identifier_from_prng,
 )
-from raiden.constants import MAXIMUM_PENDING_TRANSFERS
+from raiden.transfer.state_change import Block
+from raiden.utils import typing
+
+
+def handle_block(
+        initiator_state: InitiatorTransferState,
+        state_change: Block,
+        channel_state: NettingChannelState,
+        pseudo_random_generator: random.Random,
+) -> TransitionResult:
+    secrethash = initiator_state.transfer.lock.secrethash
+    locked_lock = channel_state.our_state.secrethashes_to_lockedlocks.get(secrethash)
+
+    if not locked_lock:
+        # Lock still valid
+        return TransitionResult(initiator_state, list())
+
+    if not channel.is_lock_expired(locked_lock, secrethash, state_change.block_number):
```

Comment:
cosmetic change.

why don't you use the same construct you have in the mediator.py?

```python
if locked_lock and channel.is_lock_expired(locked_lock, secrethash, block_number):
```